src/org/opensolaris/opengrok/configuration/messages/ConfigMessage.java

```diff
@@ -209,6 +211,53 @@ protected void invokeSetter(Configuration config, String field, String value) th
         }
     }
 
+    /**
+     * Invokes a getter of a property on the configuration object.
+     * 
+     * @param config configuration object
+     * @param field string with field name
+     * @return string representation of the field value
+     * @throws java.io.IOException 
+     */
+    protected String invokeGetter(Configuration config, String field) throws IOException {
```

Comment:
this may be a candidate for a util function along with invokeSetter

Reply:
you mean moving this to `org.opensolaris.opengrok.util` as a utility class ? or possibly to `Configuration.java` ?

Reply:
t thought more about a new utility class but you decide both is better than have it here

Reply:
or maybe if you extract this to the ClassUtils and make it more generic (for any object and not just for configuration) and then you can delegate these calls from here. How about that?

Reply:
yep, that's what I did.